test/common/testinfra.py

```diff
@@ -46,7 +46,7 @@
 WHITELIST_LOCAL = "~/.config/github-whitelist"
 
 HOSTNAME = socket.gethostname().split(".")[0]
-DEFAULT_IMAGE = os.environ.get("TEST_OS", "fedora-25")
+DEFAULT_IMAGE = os.environ.get("TEST_OS", "fedora-26")
```

Comment:
Agreed. As a first step, let's get it running in CI regularly and flip over at beta. At that point we should all upgrade our laptops to F26 beta anyway :-)